ElementX/Sources/Services/Audio/Player/AudioPlayerState.swift

```diff
@@ -200,6 +214,93 @@ class AudioPlayerState: ObservableObject, Identifiable {
             .removeDuplicates()
             .weakAssign(to: \.playerButtonPlaybackState, on: self)
     }
+    
+    private func setUpRemoteCommandCenter() {
+        UIApplication.shared.beginReceivingRemoteControlEvents()
+        
+        let commandCenter = MPRemoteCommandCenter.shared()
+        
+        commandCenter.playCommand.isEnabled = true
+        commandCenter.playCommand.removeTarget(nil)
+        commandCenter.playCommand.addTarget { [weak self] _ in
+            guard let audioPlayer = self?.audioPlayer else {
+                return MPRemoteCommandHandlerStatus.commandFailed
```

Comment:
Can these all be simplified to?
```suggestion
                return .commandFailed
```